developer-experience/recipes/remote-debugging/kubernetes-dashboard/recipes/setup-kubernetes-dashboard.md

```diff
@@ -0,0 +1,51 @@
+# Kubernetes Dashboard
+
+## Background
+
+The Kubernetes dashboard is a web-based UI for Kubernetes which provides an overview of the Kubernetes cluster as well as the individual resources running in it. Viewing, monitoring, and debugging operational aspects of the cluster is valuable in order to understand all the components and where the issues may exist during the debugging process.
+
+## General Setup
+
+1. [Install the Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli)
+
+2. If you do not already have one, [follow these steps to create an AKS Cluster](https://docs.microsoft.com/en-us/azure/aks/kubernetes-walkthrough)
+
+3. Install the command-line tool, [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/), which will allow you to run commands against the Kubernetes cluster.
+
+4. Connect to the cluster using the Azure CLI with the command:
```

Comment:
I suggest we move #4 to the next section. But I get why you kept it here.

Reply:
With the general setup the intent was to put everything up to connecting the cluster and the next section would focus on just setting up the dashboard. If you are good, I will leave it in the general section? Unless you feel it will flow better in the next section?

Reply:
makes sense.